bot/code_review_bot/config.py

```diff
@@ -16,7 +16,8 @@
 import yaml
 
 PROJECT_NAME = 'code-review-bot'
-REPO_TRY = b'https://hg.mozilla.org/try'
```

Comment:
Why was this a bytes? Sure changing it to a string doesn't break anything?

Reply:
It was bytes to be used by `hglib` to clone from that string. Since i moved to url from secrets, it's no longer used for that purpose . The only usage is now to do the check on decision task.